lib/spack/spack/cmd/common/arguments.py

```diff
@@ -437,3 +439,46 @@ def add_s3_connection_args(subparser, add_help):
     subparser.add_argument(
         "--s3-endpoint-url", help="Endpoint URL to use to connect to this S3 mirror"
     )
+
+
+def use_buildcache(opt):
```

Comment:
Can we rename it to something like:
```suggestion
def use_buildcache(cli_args):
```
?